autogpt_platform/backend/backend/blocks/test_mutual_exclusive.py

```diff
@@ -0,0 +1,74 @@
+from typing import Optional
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import SchemaField
+
+
+class TestMutualExclusiveBlock(Block):
```

Comment:
This PR is a good addition - for the tests however, I wonder if you could put the tests in one of the real blocks to show it working?